src/main/java/nl/ordina/jobcrawler/scrapers/YachtVacancyScraper.java

```diff
@@ -83,8 +85,7 @@ public YachtVacancyScraper() {
      * @param pageNumber Pagenumber of which the vacancy data should be retrieved
      * @return json response from the get request
      */
-    private YachtVacancyResponse scrapeVacancies(int pageNumber) {
-        RestTemplate restTemplate = new RestTemplate();
+    YachtVacancyResponse scrapeVacancies(int pageNumber) {
```

Comment:
Why default accessibility? I would choose explicit access modifiers, preferably public or private depending on the method's usage. You might need protected, but I would never use the default modifier.

Reply:
We've had some discussions about this. We want to test our private methods, but that isn't a possibility. Because of that we've been looking into what we can do. Private is the least visible and makes testing pretty much impossible. After that no modifier (package-private) is the least visible according to Oracle (https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html). That's the main reason why we decided to do this. Any suggestion to do this otherwise is appreciated.